### PR TITLE
examples: Add AlmaLinux to Lima 

### PR DESCRIPTION
**Pull request by reishoku**
This PR contains:
- example YAML files of AlmaLinux VM

I've confirmed on my machine that AlmaLinux works with Lima.

----------------------------
Signed-off-by: KOSHIKAWA Kenichi <reishoku.misc ~~@~~ pm.me>

### CODE REVIEW

File: examples/almalinux.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  digest: "sha512:19623e60216800fb3cd9f564c6d20415adc6d990ba4924fef2a2056035049d2ac2c749c8b71843f62af2955e8b505f4705daacb16c13d3079366b007ad846491"
+- location: "http://repo.almalinux.org/almalinux/8.5/cloud/aarch64/images/AlmaLinux-8-GenericCloud-latest.aarch64.qcow2"
+  arch: "aarch64"
+  digest: "sha512:bf8c0afaa8aaced047977de73455f09d2867907efece91cbad16dbd3ab454328b432eadccc62cdfc6e529f5147e564c9d97405a2598efde6758151c6fda758d4"
+mounts:
+- location: "~"
+- location: "/tmp/lima"
+  writable: true
+firmware:
+  legacyBIOS: true
```
Comment (AkihiroSuda) [inline examples/almalinux.yaml:15]: I guess we need this for Alma too?
https://github.com/lima-vm/lima/blob/f24302b6d0a8b5ea2666a6b5eea169d64c20ccea/examples/rocky.yaml#L3-L5

File: README.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ Alternatively, you may also directly ssh into the guest: `ssh -p 60022 -i ~/.lim
 Yes, it should work, but not regularly tested on ARM (due to lack of CI).
 
 #### "Can I run non-Ubuntu guests?"
-Alpine, Arch Linux, Debian, Fedora, openSUSE, and Rocky are also known to work.
+AlmaLinux, Alpine, Arch Linux, Debian, Fedora, openSUSE, and Rocky are also known to work.
```
Comment (AkihiroSuda) [inline README.md:251]: We have a similar list in the top of README.md too

File: README.ja.md [hunk 3]
```diff
@@ -242,7 +242,7 @@ Limaにはデータの喪失を引き起こすバグが含まれているかも
 はい。動くでしょう。ただ、ARM Macでの定期的なテストはなされていません(CIがないため)。
 
 #### "Ubuntu以外のゲストを動かすことはできますか？"
-Alpine, Arch Linux, Debian, Fedora, openSUSE, Rocky が動くことが知られています。[`./examples/`](./examples/)を見てください。
+AlmaLinux, Alpine, Arch Linux, Debian, Fedora, openSUSE, Rocky が動くことが知られています。[`./examples/`](./examples/)を見てください。
```
Comment (AkihiroSuda) [inline README.ja.md:245]: Japanese translation should be a separate commit.
Please update the revision/date in the header too.

File: README.ja.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-This is an *unofficial* translation of [`README.md` (revision fce1b2fad8, 2022-01-28)](https://github.com/lima-vm/lima/blob/fce1b2fad85762d72d2e4d37e4ab211f570fa2d1/README.md).
+This is an *unofficial* translation of [`README.md` (revision 4c7376a, 202200-05)](https://github.com/lima-vm/lima/blob/4c7376a9a7ac54514611a39ea720eb9798bfe973/README.md).
```
Comment (AkihiroSuda) [inline README.ja.md:1]: typo

**Review by AkihiroSuda — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
Bikeshedding: the filename should be `alma.yaml` or `almalinux.yaml`?

So far,  the only YAML with the `linux` suffix is `archlinux.yaml` because `arch` often means CPU archs such as `x86_64`.

**Comment by reishoku** [comment]
comment: According to [AlmaLinux's official website](https://almalinux.org), the distro name is "AlmaLinux", not "Alma Linux". That's why I used "almalinux" for filename.

**Comment by afbjorklund** [comment]
It would help with some background information on what the Linux distribution is, and what it offers...

It seems to be yet another RHEL clone, but I guess that was to be expected when CentOS died ?

> AlmaLinux OS is 1:1 binary compatible with RHEL®

> Rocky Linux is an open-source enterprise operating system designed to be 100% bug-for-bug compatible with Red Hat Enterprise Linux®.

https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives

Maybe they can be grouped or labelled or something.

NAME="AlmaLinux"
VERSION="8.5 (Arctic Sphynx)"
ID="almalinux"
**ID_LIKE="rhel centos fedora"**
VERSION_ID="8.5"
PLATFORM_ID="platform:el8"
PRETTY_NAME="AlmaLinux 8.5 (Arctic Sphynx)"

NAME="Rocky Linux"
VERSION="8.5 (Green Obsidian)"
ID="rocky"
**ID_LIKE="rhel centos fedora"**
VERSION_ID="8.5"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.5 (Green Obsidian)"

I would expect them to behave "similar".

**Comment by afbjorklund** [comment]
Side note:
Confusingly, both CentOS 8.5 and CentOS 8-Stream have the **same** identifiers.

NAME="CentOS Linux"
VERSION="8"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8"

NAME="CentOS Stream"
VERSION="8"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Stream 8"

This is going to be lots of fun, trying to sort those out. So lima didn't even try...

**Comment by jandubois** [comment]
> It seems to be yet another RHEL clone, but I guess that was to be expected when CentOS died ?

Yes, they are both trying to be the replacement for CentOS. Rocky is headed by the original creator of CentOS while AlmaLinux is under control of a foundation created (and sponsored) by CloudLinux.

They both want to be the same thing, and they should be extremely similar by design.

**Review by jandubois — COMMENTED** [review]
Seems fine to me, but there are unaddressed comments from @AkihiroSuda. Not sure what to make of this, as the PR was approved after making those comments; so not sure if they should still be addressed or not.

**Comment by afbjorklund** [comment]
> They both want to be the same thing, and they should be extremely similar by design.

Fedora LTS 😀

I made some table in some other context, but it should be very similar w.r.t. the RPMS.

Fedora | CentOS |
------ | ------ |
fc12   | el6    |
fc18   | el7   |
fc28   | el8    |
fc34   | el9    |

**Review by AkihiroSuda — APPROVED** [review]

**Comment by afbjorklund** [comment]
Wonder why the alma image is so much smaller than the rocky (and centos) image ?

```
file format: qcow2
virtual size: 10 GiB (10737418240 bytes)
disk size: 530 MiB
cluster_size: 65536
```

```
file format: qcow2
virtual size: 10 GiB (10737418240 bytes)
disk size: 1.36 GiB
cluster_size: 65536
```

Almost a GB in difference, but the compression and the rpm package list look similar ?

**Comment by afbjorklund** [comment]
> According to [AlmaLinux's official website](https://almalinux.org), the distro name is "AlmaLinux", not "Alma Linux"

Most ridiculous thiing I have heard, but I guess they "inherited" it from CloudLinux (without space).
